src/chart_types/xy_chart/renderer/canvas/xy_chart.tsx

```diff
@@ -172,8 +182,18 @@ class XYChartComponent extends React.Component<XYChartProps> {
     const chartSeriesTypes =
       seriesTypes.size > 1 ? `Mixed chart: ${[...seriesTypes].join(' and ')} chart` : `${[...seriesTypes]} chart`;
     const chartIdDescription = `${chartId}--description`;
+    const chartIdLabel = ariaLabel ? `${chartId}--label` : undefined;
+
+    const ariaProps: AriaProps = {};
+
+    if (ariaLabelledBy || ariaLabel) {
+      ariaProps['aria-labelledby'] = ariaLabelledBy ?? chartIdLabel;
+    }
+    if (ariaLabelledBy || useDefaultSummary) {
+      ariaProps['aria-describedby'] = `${ariaLabelledBy || ''} ${useDefaultSummary ? chartIdLabel : undefined}`;
+    }
```

Comment:
@markov00 Good catch in https://github.com/elastic/elastic-charts/pull/1121#discussion_r616829738

Adding a suggestion here so I could target more lines, I think the `ariaLabelledBy` stuff snuck in by mistake. It should probably be something like:

```suggestion
    if (accessibilityDescription || useDefaultSummary) {
      ariaProps['aria-describedby'] = `${accessibilityDescription ? chartIdDescription : undefined} ${useDefaultSummary ? aNewIdForTheChartSeriesTypesDD : undefined}`;
    }
```

(I'm suggesting making a new id just for the `chartSeriesTypes` `<dd>` because description text can't be structured data so we can't just set an id for the whole `<dl>` and use that. It's not perfect but it works well enough for a quick description imo. 

@rshen91 Feel free to reach out if this suggestion doesn't make any sense! It's a little convoluted.

Reply:
Woops added this in 6150d2d thank you!